### PR TITLE
chore: adding bucket tests

### LINKED ISSUE #2454: Bucket.put SDK spec tests

### LINKED ISSUE #2456: Bucket.get SDK spec tests

### LINKED ISSUE #2457: put_json

### LINKED ISSUE #2458: get_json

### PR DESCRIPTION
**Pull request by revitalbarletz**
fixes  #2454
fixes #2457
fixes #2456
fixes #2458

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Comment by eladb** [comment]
@revitalbarletz PR title needs to be fixed

**Review by eladb — APPROVED** [review]

File: examples/tests/sdk_tests/bucket/put_json.w [hunk 1]
```diff
@@ -0,0 +1,24 @@
+bring cloud;
+
+let b = new cloud.Bucket();
+
+new cloud.Function(inflight () => {
+
+  let json_obj1 = Json { test: "test1" };
+  let json_obj2 = Json { test: "test2" };
+
+  b.put_json("test1.txt", json_obj1);
```
Comment (tsuf239) [inline examples/tests/sdk_tests/bucket/put_json.w:10]: I think the right suffix would be `.json`

File: examples/tests/sdk_tests/bucket/put_json.w [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  b.put_json("test2.txt", json_obj2);
+
+  let test_json1 = b.get_json("test1.txt");
+  let test_json2 = b.get_json("test2.txt");
+  
+  assert(test_json1.get("test1") == json_obj1.get("test1"));
```
Comment (tsuf239) [inline examples/tests/sdk_tests/bucket/put_json.w:16]:
```suggestion
  assert(test_json1.get("test") == json_obj1.get("test"));
```

File: examples/tests/sdk_tests/bucket/put_json.w [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  assert(test_json2.get("test2") == json_obj2.get("test2"));
```
Comment (tsuf239) [inline examples/tests/sdk_tests/bucket/put_json.w:17]:
```suggestion
  assert(test_json2.get("test") == json_obj2.get("test"));
```

File: examples/tests/sdk_tests/bucket/put_json.w [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  let json_obj3 = Json { test: "test3" };
+  b.put_json("test3.txt", json_obj3);
+  let test_json3 = b.get_json("test3.txt");
+  assert(test_json1.get("test3") == json_obj1.get("test3"));
```
Comment (tsuf239) [inline examples/tests/sdk_tests/bucket/put_json.w:22]:
```suggestion
  assert(test_json3.get("test") == json_obj3.get("test"));
```

File: examples/tests/sdk_tests/bucket/put_json.w [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (tsuf239) [inline examples/tests/sdk_tests/bucket/put_json.w:23]: maybe add a deletion too?

**Review by tsuf239 — COMMENTED** [review]
Really great job! thank you for the help!!
can you please run `npx nx hangar test` to generate snapshots, and then `npm run build`?

File: examples/tests/sdk_tests/bucket/put_json.w (hunk 5 above)
Comment (revitalbarletz) [reply examples/tests/sdk_tests/bucket/put_json.w:23]: Done.

**Pull request merged**